resources/ProductResource.json

```
@@ -0,0 +1,74 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "/resources/ProductResource",
+    "type": "object",
+    "allOf": [
+        {"$ref": "../types/RelationType.json"}
+    ],
+    "properties": {
+        "type": {
+            "$ref": "../enums/productType.json"            
+        },
+        "manufacturer": {
+            "type": "string"
+        },
+        "brand": {
+            "type": "string"
+        },
+        "form": {
+            "$ref": "../enums/productForm.json"            
+        },
+        "uri": {
+            "type": "string",
+            "format": "uri-reference",
+            "description": "URL to the product."
+          },
+        "crop" : {
+            "$ref": "../types/CropType.json"
+        },
+        "specificGravity": {
+            "type":"number"
```

Possible description?

```suggestion
            "type": "number",
            "description": "Relative density. The ratio of the density (mass of a unit volume) of a substance to the density of a given reference material. For liquids this is typically water at 4 degrees celcius"
```